### ISSUE TITLE
Learning rate difference.

### ISSUE THREAD

**Issue by korkinof** [shanexn/pytorch-pggan#2]
Any reason the learning rate is set 10x lower than the original TF implementation?
Did it work with this lr and not the original?
I was wondering of any underlying implementation differences of PyTorch that may make it tricky to reproduce the results.
